cli/electron.ts

```diff
@@ -372,7 +372,7 @@ function electronGulpTask(taskName: string): Promise<void> {
     const gulpPath = path.join(pxtElectronPath, "node_modules", ".bin", "gulp");
 
     return nodeutil.spawnAsync({
-        cmd: nodeutil.addCmd(gulpPath),
+        cmd: "gulp.cmd",
```

Comment:
Before shifting to global gulp, can you try something like:
```
cmd: `"${nodeutil.addCmd(gulpPath)}"`
```
Thanks for looking into this!